File: Learn_Python_the_Hard_Way/ex5.py
name = 'Zed A. Shaw'
age = 35 #not a lie in 2009
height = 74 #inches
metHeight = height * 2.54
weight = 180 #pounds
metWeight = weight * 0.453592
eyes = 'Blue'
teeth = 'White'
hair = 'Brown'

# the ("%s" % my_var) is an embedded variable in the string. 
# If you just add the variable in maually between strings, python concats.
print "Let's talk about %s." % name
print "He's %d inches tall, which is %d centemeters." % (height, round(metHeight))
print "He's %d pounds heavy, which is %d kilograms." % (weight, round(metWeight))
print "Actually that's not too heavy."
print "He's got %s eyes and %s hair." % (eyes, hair)
print "His teeth are usually %s depending on the coffee." % teeth
print "If I add %d, %d, and %d I get %d" % (age, height, weight, age + height + weight)

File: LRtHW/ex5/ex5.rb
class Ex5

  def variables

    my_name = 'Zed A. Shaw'
    my_age = 35 # not a lie in 2009
    my_height = 74 # inches
    my_weight = 180 # lbs
    my_eyes = 'Blue'
    my_teeth = 'White'
    my_hair = 'Brown'

    puts "Let's talk about #{my_name}."
    puts "They're #{my_height} inches tall."
    puts "They're #{my_weight} pounds heavy."
    puts "Actually that's not too heavy."
    puts "They've got #{my_eyes} eyes and #{my_hair} hair."
    puts "Their teeth are usually #{my_teeth} depending on the coffee."

    # this line is tricky, try to get it exactly right
    puts "If I add #{my_age}, #{my_height}, and #{my_weight} I get #{my_age + my_height + my_weight}."

  end 

end

ex5 = Ex5.new()

ex5.variables()
